src/universal/components/RetroDiscussPhase.js

```diff
@@ -140,45 +156,59 @@ const RetroDiscussPhase = (props: Props) => {
   return (
     <React.Fragment>
       <PhaseWrapper>
-        <ReflectionSection>
-          <ScrollableBlock>
-            <ReflectionSectionInner>
-              <DiscussHeader>
-                <TopicHeading>
-                  <span>{'“'}</span>
-                  {`${title}”`}
-                </TopicHeading>
-                <VoteMeta>
-                  <VoteIcon name={meetingVoteIcon} />
-                  {voteCount}
-                </VoteMeta>
-              </DiscussHeader>
-              <ReflectionGrid>
-                {reflections.map((reflection) => {
-                  return (
-                    <ReflectionCard
-                      key={reflection.id}
-                      meeting={newMeeting}
-                      reflection={reflection}
-                    />
-                  )
-                })}
-              </ReflectionGrid>
-            </ReflectionSectionInner>
-          </ScrollableBlock>
-        </ReflectionSection>
-        <TaskSection>
-          <ScrollableBlock>
-            <TaskCardBlock>
-              <MeetingAgendaCards
-                meetingId={meetingId}
-                reflectionGroupId={reflectionGroupId}
-                tasks={tasks}
-                teamId={teamId}
-              />
-            </TaskCardBlock>
-          </ScrollableBlock>
-        </TaskSection>
+        <HeaderContainer>
+          <DiscussHeader>
+            <TopicHeading>
+              <span>{'“'}</span>
+              {`${title}”`}
+            </TopicHeading>
+            <VoteMeta>
+              <VoteIcon name={meetingVoteIcon} />
+              {voteCount}
+            </VoteMeta>
+          </DiscussHeader>
+        </HeaderContainer>
+        <ColumnsContainer>
+          <Column>
+            <LabelContainer>
+              <LabelHeading>
+                {reflections.length} {plural(reflections.length, 'Reflection')}
+              </LabelHeading>
+            </LabelContainer>
+            <ScrollableBlock hideOverflownShadow>
+              <ColumnInner>
+                <ReflectionGrid>
+                  {reflections.map((reflection) => {
+                    return (
+                      <ReflectionCard
+                        key={reflection.id}
+                        meeting={newMeeting}
+                        reflection={reflection}
+                      />
+                    )
+                  })}
+                </ReflectionGrid>
+              </ColumnInner>
+            </ScrollableBlock>
+          </Column>
+          <TaskColumn>
+            <LabelContainer>
+              <LabelHeading>Takeaway Tasks</LabelHeading>
+            </LabelContainer>
+            <ScrollableBlock hideOverflownShadow>
```

Comment:
can we use a new component instead of ScrollableBlock? 
- even with the new prop, it still calculates the shadow, which is really expensive
- with shadows, it gives lists a sunken in look, which isn't great
- the component just needs to die a slow fiery death

Reply:
Very good points! Will do. 👍